test/infrastructure/docker/templates/clusterclass-quick-start.yaml

```diff
@@ -94,44 +94,6 @@ spec:
         path: "/spec/template/spec/kubeadmConfigSpec/clusterConfiguration/imageRepository"
         valueFrom:
           variable: imageRepository
```

Comment:
Diffed against main. Should we also drop the replaceImage patches?

Reply:
I would say that the patch changing the image repository in clusterConfiguration is orthogonal on this issue; also, if we want to change something here, we have to change it in main first

Reply:
We changed it on main already. I found this by diffing against main

Reply:
For reference: https://github.com/kubernetes-sigs/cluster-api/pull/8921/files#diff-813c2a40b85d2d2a6737e5f20041314716efaf113af26c458f50912f764fa4a4

Reply:
I think I've removed the patch you were talking about now - let me know if I missed something.

Reply:
Yup, looks perfect!